tony-creating-song-about-vertcoin.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: fr
+title: Creating a song about Vertcoin
+author: Koresh Tony
+date: February 15, 2023
+amount: 7750
+milestones:
+  - name: Milestone 1 - Writing lyrics
+    funds: 50% (3875 VTC)
+    done:
+    status: unfinished
+  - name: Milestone 2 - Song recording
+    funds: 50% (3875 VTC)
+    done:
+    status: unfinished
+payouts:
+  - date:
+    amount:
+  - date:
+    amount:
+---
+Track duration ~ 3minutes.
```

Comment:
Could you please copy this entire description into the comment field on the pull request so it's visible?